src/Core/src/Handlers/Editor/EditorHandler.Android.cs

```diff
@@ -45,13 +45,15 @@ protected override void ConnectHandler(AppCompatEditText platformView)
 		{
 			platformView.ViewAttachedToWindow += OnPlatformViewAttachedToWindow;
 			platformView.TextChanged += OnTextChanged;
+			platformView.SetOnTouchListener(new EditorTouchListener());
```

Comment:
What happens if you are using gestures on this control? Does the `SetOnTouchListener` overwrite that? 

I wonder if there is some event we should be using instead? Maybe a long press or pointer gesture. I am not 100% sure of this, but I think the gestures also use the Touch event, which means that this will overwrite it. You may also need to use the event as opposed to this listener. 

But, this is more a gut feeling than knowledge. @PureWeen may have a "yes Mathew, you are smart" or "wow, I can't believe I work with that guy... Do you even code bro? Give him some of the I from AI" :)